### PR TITLE
Fix MapTypeStyle, typo

### PR DESCRIPTION
**Pull request by trejjam**
Description MapTypeStylers: https://developers.google.com/maps/documentation/javascript/styling#map_features

### CODE REVIEW

File: googlemaps/google.maps.d.ts [hunk 1]
```diff
@@ -1204,58 +1204,11 @@ declare module google.maps {
     }
 
     export interface MapTypeStyle {
-        elementType?: MapTypeStyleElementType;
-        featureType?: MapTypeStyleFeatureType;
+        elementType?: string;
```
Comment (horiuchi) [inline googlemaps/google.maps.d.ts]: I thought this type can be `string|MapTypeStyleElementType`.
Next line too.

Comment (Trejjam) [inline googlemaps/google.maps.d.ts]: I don't think it is neccessary, but ok.

Comment (horiuchi) [inline googlemaps/google.maps.d.ts]: Thank you!

**Comment by horiuchi** [comment]
@Trejjam Thanks mate!

**Pull request merged**